providers/amazon/src/airflow/providers/amazon/aws/operators/ssm.py

```diff
@@ -90,7 +90,7 @@ def execute_complete(self, context: Context, event: dict[str, Any] | None = None
         event = validate_execute_complete_event(event)
 
         if event["status"] != "success":
-            raise AirflowException(f"Error while running run command: {event}")
+            raise RuntimeError(f"Error while running run command: {event}")
```

Comment:
Was there any other code that was catching the previous AriflowException that now needs to be updated to catch the RuntimeException?